wiki/Rules/en.md

```diff
@@ -1,77 +1,81 @@
+3.  **Be productive with your criticism without resorting to personal attacks.** Criticism is a wonderful thing when done properly, but if you're resorting to personal attacks to make your point, you're doing it wrong, and you should feel bad.
+4.  **Keep threads relevant, and avoid resurrecting older topics if the conversation has died out.** If a topic has been dead for years, it isn't really relevant anymore, so just let it fade away. Post a new one, or maybe search and see if someone has answered your question already.
+5.  **Keep your signatures and avatars reasonable.** If they're animated, they shouldn't strobe or flash excessively, and they should be of a filesize small enough for users with slower connections to not feel their presence on the page.
+6.  **Don't post advertisements, referral links or any other annoying or intrusive content.** Things like affiliate and referral links are fun for the person posting them and an annoyance for everybody else, so don't do it.  Autoloading or autoplaying content/embeds falls within this rule as well.
+7.  **Linking to applications or utilities of uncertain intent is not allowed.** Put simply, if there is any doubt as to the legality of said utilities, they are not welcome to be posted.
+
+Beatmap Submission Rules
 ---------------------------------
 
-These rules apply strictly to the unique nature of the [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. These threads are created via the osu! client when a beatmap is submitted through the [editor](/wiki/Beatmap_Editor).
+These rules apply strictly to beatmaps submitted via the in-game [Beatmap Submission System](/wiki/Beatmap_Submission_System). These threads/discussions are created via the osu! client when a beatmap is submitted through the [editor](/wiki/editor).
 
-1.  Do not create threads in the [Pending Submissions](/wiki/Beatmaps) and [WIP](/wiki/Glossary) boards. Use the appropriate subforum when creating a thread. (This obviously excludes BSS threads.)
-2.  Do not get off topic in BSS threads. BSS threads are about the parent beatmap. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
-3.  Do not steal others' beatmaps, presenting them as if they're mapped by you. Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
+A set of more detailed rules for beatmap construction exists, known as the [Ranking Criteria](/wiki/Ranking_Criteria).
+
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hitsounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in osu!.
```

Comment:
the ! in osu! is part of the identifying semantic object and thus does not fill its role as an exclamation point, so a full-stop is grammatically accurate here.